style.css

```diff
@@ -323,3 +324,69 @@ main {
 .drop {
   color: #abc7c3;
 }
+
+.mobile-footer {
+  display: block;
+  margin-top: 30px;
+  background: #6070ff;
+  border-radius: 100px 0 0 0;
+  background-image: url('images/contact form shapes-mobile.svg');
+  background-repeat: no-repeat;
+  background-position: 100%;
+  padding-block: 8.5rem 4rem;
+  padding-inline: 5%;
+}
+
+.form-title {
+  text-align: center;
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.mobile-footer p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+}
+
+.mobile-form {
+  display: block;
+  margin-top: 30px;
+}
```

Comment:
- Great Job for trying to stick the Figma design, but if you used `flexbox` your style will be easy not only for this mobile version it will help you also to the desktop version, so please use `flexbox` it is also mentioned in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)👌👍